serving-tiles/manually-building-a-tile-server-16-04-2-lts.md

```diff
@@ -101,6 +101,13 @@ Again, say yes to install.
 
     sudo make install
 
+## Postgresql optimization
+
+If you have a default installation of postgresql it is very unlikely that it will be configured to offer good performance by default.
```

Comment:
```suggestion
If you have a default installation of PostgreSQL it is very unlikely that it will be configured to offer good performance by default.
```